### ISSUE TITLE
[Bug]: [1.4.2] Conditions are not working 

### ISSUE THREAD

**Issue by Wirless** [otland/forgottenserver#4813]
### By submitting this bug issue, you agree to the following.

- [X] This is a bug in the software that resides in this repository, and not a support matter (use https://otland.net/forums/support.16/ for support)
- [X] This issue is reproducible without changes to the C++ code in this repository
- [X] This bug has not been resolved in master branch
- [X] There is no existing issue for this bug already

### Does this bug crash tfs?

no

### Server Version

1.4.x (Release)

### Operation System

all (listed below)

### Bug description

Damage is only dealt on players combat.

### Possible Pull Requests which are to blame

- [ ] #
- [ ] #

### Steps to reproduce

1.
2.
3.
**Code to reproduce the issue:**
```lua
-- paste your script here
```

### Actual Behavior

When player casts spell which should give condition the condition does apply but never ticks
the only time u get a tick if player (owner of condition) deals the damage to the monster.
e.g try using poison arrow itself it does not actually work dealing the poison ticks over time no matter what way you add condition
I tried 50 different ways to force condition create second combat object and everything 
issue possibly comes from Updated lua libraries or some other stuff while it might have worked in 2018-2019 it does in fact not work nowadays.

### Expected Behavior

Conditions should tick in their own respected time
use poison arrow for reference.
there is never a second tick ever unless player deals the damage to monster via weapon/spell/ or other combat only then it ticks and player has to finely TIME his hit with the tick otherwise it wont tick either.

**Comment by Wirless** [comment]
seems fine for the fluids however but thats on self which is odd

**Comment by Wirless** [comment]
i could not find any reference to the poison arrow addConditionDamage or whatever htat function is anywhere in luascripts

**Comment by ArturKnopik** [comment]
can you provided some example script? 

probably this is source of problems:
https://github.com/otland/forgottenserver/blob/3bca716b7a547a5fb78b73483c802b8e523bd8c0/src/combat.cpp#L677

**Comment by nok1111** [comment]
seems like it happens when u apply a dondition to combat.

**Comment by nekiro** [comment]
Provide example to reproduce

**Comment by Wirless** [comment]
Poison arrow in weapons

**Comment by Wirless** [comment]
yeah conditions applied in combat only tick if the combat is ticking if there is no reaction to combat then the condition time is not running or its running but skipping the tick adding  yoiu have to hit monster at either extactly the tick time or near it for the tick to happen

**Comment by theshibbies** [comment]
Adding CONDITION_PARAM_AGGRESSIVE to Creature:addDamageCondition worked for me
